<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>32</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>思源黑体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Space</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/systemTrayIcon/res/icon.png</normaloff>:/systemTrayIcon/res/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(233, 237, 244);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="code_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{border-image: url(:/icon/res/btn_icon/hover/code.png)}
QPushButton:pressed{border-image: url(:/icon/res/btn_icon/press/code.png)}
QPushButton{border-image: url(:/icon/res/btn_icon/default/code.png)}
QPushButton:!enabled{border-image: url(:/icon/res/btn_icon/unable/code.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="latex_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/icon/res/btn_icon/default/LaTex.png)}
QPushButton:hover{border-image: url(:/icon/res/btn_icon/hover/LaTex.png)}
QPushButton:pressed{border-image: url(:/icon/res/btn_icon/press/LaTex.png)}
QPushButton:!enabled{border-image: url(:/icon/res/btn_icon/unable/LaTex.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ocr_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/icon/res/btn_icon/default/OCR.png)}
QPushButton:hover{border-image: url(:/icon/res/btn_icon/hover/OCR.png)}
QPushButton:pressed{border-image: url(:/icon/res/btn_icon/press/OCR.png)}
QPushButton:!enabled{border-image: url(:/icon/res/btn_icon/unable/OCR.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="edit_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/icon/res/btn_icon/default/edit.png)}
QPushButton:hover{border-image: url(:/icon/res/btn_icon/hover/edit.png)}
QPushButton:pressed{border-image: url(:/icon/res/btn_icon/press/edit.png)}
QPushButton:!enabled{border-image: url(:/icon/res/btn_icon/unable/edit.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="concat_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/icon/res/btn_icon/default/layout.png)}
QPushButton:hover{border-image: url(:/icon/res/btn_icon/hover/layout.png)}
QPushButton:pressed{border-image: url(:/icon/res/btn_icon/press/layout.png)}
QPushButton:!enabled{border-image: url(:/icon/res/btn_icon/unable/layout.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setting_button">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/icon/res/btn_icon/default/adjust.png)}
QPushButton:hover{border-image: url(:/icon/res/btn_icon/hover/adjust.png)}
QPushButton:pressed{border-image: url(:/icon/res/btn_icon/press/adjust.png)}
QPushButton:!enabled{border-image: url(:/icon/res/btn_icon/unable/adjust.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="back_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/icon/res/btn_icon/default/back.png)}
QPushButton:hover{border-image: url(:/icon/res/btn_icon/hover/back.png)}
QPushButton:pressed{border-image: url(:/icon/res/btn_icon/press/back.png)}
QPushButton:!enabled{border-image: url(:/icon/res/btn_icon/unable/back.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="init_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/icon/res/btn_icon/default/refresh.png)}
QPushButton:hover{border-image: url(:/icon/res/btn_icon/hover/refresh.png)}
QPushButton:pressed{border-image: url(:/icon/res/btn_icon/press/refresh.png)}
QPushButton:!enabled{border-image: url(:/icon/res/btn_icon/unable/refresh.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="source_button">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/icon/res/btn_icon/default/picture.png)}
QPushButton:hover{border-image: url(:/icon/res/btn_icon/hover/picture.png)}
QPushButton:pressed{border-image: url(:/icon/res/btn_icon/press/picture.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pin_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/icon/res/btn_icon/default/pin.png)}
QPushButton:hover{border-image: url(:/icon/res/btn_icon/hover/pin.png)}
QPushButton:pressed{border-image: url(:/icon/res/btn_icon/press/pin.png)}
QPushButton:!enabled{border-image: url(:/icon/res/btn_icon/unable/pin.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exit_button">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/icon/res/btn_icon/default/close.png)}
QPushButton:hover{border-image: url(:/icon/res/btn_icon/hover/close.png)}
QPushButton:pressed{border-image: url(:/icon/res/btn_icon/press/close.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="shortcut">
         <string>Esc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="export_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/icon/res/btn_icon/default/export.png)}
QPushButton:hover{border-image: url(:/icon/res/btn_icon/hover/export.png)}
QPushButton:pressed{border-image: url(:/icon/res/btn_icon/press/export.png)}
QPushButton:!enabled{border-image: url(:/icon/res/btn_icon/unable/export.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save_button">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/icon/res/btn_icon/default/download.png)}
QPushButton:hover{border-image: url(:/icon/res/btn_icon/hover/download.png)}
QPushButton:pressed{border-image: url(:/icon/res/btn_icon/press/download.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copy_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-image: url(:/icon/res/btn_icon/default/copy.png)}
QPushButton:hover{border-image: url(:/icon/res/btn_icon/hover/copy.png)}
QPushButton:pressed{border-image: url(:/icon/res/btn_icon/press/copy.png)}
QPushButton:!enabled{border-image: url(:/icon/res/btn_icon/unable/copy.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
